homeassistant/components/avea/light.py

```diff
@@ -0,0 +1,92 @@
+import logging
+from random import randint
+import voluptuous as vol
+
+from homeassistant.components.light import ATTR_BRIGHTNESS, ATTR_HS_COLOR, ATTR_RGB_COLOR, SUPPORT_BRIGHTNESS, SUPPORT_COLOR, Light, PLATFORM_SCHEMA
+import homeassistant.helpers.config_validation as cv
+import homeassistant.util.color as color_util
+
+
+_LOGGER = logging.getLogger(__name__)
+
+SUPPORT_AVEA = (SUPPORT_BRIGHTNESS | SUPPORT_COLOR)
+
+
+def setup_platform(hass, config, add_devices, discovery_info=None):
+    """Setup the Avea platform."""
+    import avea
+
+    nearbyBulbs = avea.discover_avea_bulbs()
+
+    add_devices(AveaLight(bulb) for bulb in nearbyBulbs)
+
+class AveaLight(Light):
+    """Representation of an Avea."""
+
+    def __init__(self, light):
+        """Initialize an AveaLigh."""
+        self._light = light
+        self._name = light.get_name()
+        if int(light.get_brightness()) == 0:
+          self._state = False
+        else:
+          self._state = True
+        self._brightness = round(255 * (light.get_brightness() / 100))
+    
+    @property
+    def supported_features(self):
+        """Flag supported features."""
+        return SUPPORT_AVEA
+
+    @property
+    def name(self):
+        """Return the display name of this light."""
+        return self._name
+
+    @property
+    def brightness(self):
+        """Return the brightness of the light.
+
+        This method is optional. Removing it indicates to Home Assistant
+        that brightness is not supported for this light.
```

Comment:
Technically not correct (I think) - supported_features indicates if a feature is supported or not.

Reply:
Just keep the first sentence. Remove this latter paragraph.

Reply:
See above. Last fix remaining.

Reply:
Damn, I missed one. Fixed in df98fc48f348593c29433d92fca39bbd5b383415 :)